test/cpp/end2end/xds/xds_override_host_end2end_test.cc

```diff
@@ -153,6 +154,12 @@ class OverrideHostTest : public XdsEnd2endTest {
     return {};
   }
 
+  static std::pair<std::string, std::string> BuildCookieHeader(
```

Comment:
This could be a method on the `Cookie` struct.

Reply:
Right! Done.